### PR TITLE
Auto save forms

### LINKED ISSUE #291: Connect Google docs' output (where drivers register) to the admin tool. OR Develop driver registration on app  TBD
Currently, drivers are registering on Google Docs.
The operators need to copy all the data to the admin manually.
This is time-consuming and creates human error.

The output of the Google Docs should automatically create a driver in the system in "draft" status.
If the email already exists in the system, it should show an error message in the form created in the admin.

Figma:
https://www.figma.com/file/3UBN2ek01pmm2muttlrxYj?node-id=448:57254&mode=design#613068382

### PR DESCRIPTION
**Pull request by yosi-miller**
closes #291 

to complete the development
1 - You need to update the app script code of the original form
2 - Update the ID of the Excel file that saves the results of the form
3 - Change the link to the global API request.

### CODE REVIEW

File: backend/src/app.ts [hunk 1]
```diff
@@ -34,7 +36,9 @@ if (config.env !== 'production') {
 app.use('/env', envRouter);
 app.use('/users', authHandler(), usersRouter);
 app.use('/drivers', authHandler(), driversRouter);
+// app.use('/signup', signupDriversRoutes);
 app.use('/rides', ridesRouter);
```
[the remark this answers is not in the document]
Comment (yosi-miller) [reply backend/src/app.ts:40]: fix

File: backend/src/server.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ async function onListening() {
   const addr = server.address();
   const bind = typeof addr === 'string' ? `pipe ${addr}` : `port ${addr.port}`;
   await connect();
-  console.log(`Listening on ${bind}`);
+  console.log(`http://localhost: on ${bind}`);
 }
```
[the remark this answers is not in the document]
Comment (yosi-miller) [reply backend/src/server.ts:36]: Because when you need the whole URL to check with Postman

**Comment by yosi-miller** [comment]
test

**Review by yitshak — APPROVED** [review]
LGTM

**Comment by yosi-miller** [comment]
Is it merged to the main?

**Pull request merged**